app/serializers/category_serializer.rb

```diff
@@ -37,6 +38,11 @@ def can_delete
     true
   end
 
+  def is_special
+    [SiteSetting.lounge_category_id, SiteSetting.meta_category_id, SiteSetting.staff_category_id, SiteSetting.uncategorized_category_id]
+    .include? object.id
+  end
```

Comment:
One thing I prefer to do to keep JSON size down is have boolean values return true always, then use `include_is_special?` with the logic. That way the attribute will not be present unless true.